packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/chat_wrappers.py

```diff
@@ -270,6 +272,8 @@ def _handle_response(
 
     # span attributes
     _set_response_attributes(span, response_dict)
+    # prompt_filter_results
+    _log_prompt_filter(span, response_dict)
```

Comment:
why is this a separate method and not part of `_set_response_attributes`?

Reply:
🤣 Just realize they're like twins. Moved it into _set_response_attributes() in __init__